packs/credential_security.yml

```diff
@@ -0,0 +1,51 @@
+AnalysisType: pack
+PackID: PantherManaged.CredentialSecurity
+Description: Detect compromised security credentials
+DisplayName: "Panther Credential Security Pack"
+PackDefinition:
+  IDs:
+    # Data Models
+    - Standard.Atlassian.Audit
+    - Standard.Github.Audit
+    - Standard.Okta.SystemLog
+    - Standard.Zendesk.AuditLog
+    # Global Helpers
+    - global_filter_auth0
+    - global_filter_github
+    - panther_auth0_helpers
+    - panther_base_helpers
+    - panther_default
+    - panther_event_type_helpers
+    # Rules
+    - Auth0.MFA.Policy.Disabled
+    - Auth0.MFA.Risk.Assessment.Disabled
+    - Asana.Workspace.Password.Requirements.Simple
+    - Dropbox.User.Disabled.2FA 
+    - Duo.Admin.App.Integration.Secret.Key.Viewed
+    - Duo.Admin.Bypass.Code.Created
+    - Duo.Admin.Bypass.Code.Viewed
+    - GitHub.Secret.Scanning.Alert.Created
+    - GSuite.LeakedPassword
+    - GSuite.Workspace.PasswordEnforceStrongDisabled
+    - GSuite.Workspace.PasswordReuseEnabled
+    - Microsoft365.MFA.Disabled
+    - Okta.Global.MFA.Disabled
+    - Okta.PasswordAccess
+    - Okta.Support.Reset
+    - Okta.User.MFA.Reset.Single
+    - Okta.User.MFA.Reset.All
+    - OneLogin.PasswordAccess
+    - OneLogin.PasswordChanged
+    - OneLogin.AuthFactorRemoved
+    - OnePassword.Lut.Sensitive.Item
+    - OnePassword.Sensitive.Item
+    - Slack.AuditLogs.MFASettingsChanged
+    - Standard.MFADisabled
+    - Zoom.Two.Factor.Authentication.Disabled
+    - AWS.Unsuccessful.MFA.attempt
```

Comment:
`AWS.Unsuccessful.MFA.attempt` is `Enabled: false`

Reply:
The other two are clearly detections that require configuration so it makes sense to me to drop them, but for this one from spot checking the rule it's not immediately clear why it's disabled by default. I checked the PR that introduced it (https://github.com/panther-labs/panther-analysis/pull/570) and I don't see any mention of it. I'll drop it for now to be safe but I'll check in with @hbenac10 on if this should be enabled generally and if so for this pack as well.